### PR TITLE
Etna and Wen's survey

### PR DESCRIPTION
**Pull request by wwenzz**
## Netlify link
https://etna-wen-project-survey.netlify.app/

## Collaborators
[Caracal23]

### CODE REVIEW

**Review by almaherris — COMMENTED** [review]
Hi Wen and Etna! 

Really good job with the project! We loved the overall vibe and the theme of your form! The code is nicely organized and flowing well! Good job 🥇 

/Alma and Linda

File: src/App.jsx [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+  const renderFinalPage = data => {
+    console.log(data);
+    const summaryText = `Hello ${data.name}! You are ${
+      data.frequency
+    } stressed. Your stress level is ${data.level}%. You have ${
+      data.waysToDeal
+    } way(s) to handle the stress. And you have subscribed to our updates on ${data.subscription.join(
+      ", "
+    )} to your email ${data.email}`;
+    //convert summary string into an array
```
[the remark this answers is not in the document]
Comment (wwenzz) [reply src/App.jsx:20]: Hey Alma, this is to separate the name and the rest of the summary text. So we can add <br /> in the SummaryPage component. If this is a string then we cannot make a new line in between in the SummaryPage.

File: src/App.jsx
Comment (AntonellaMorittu) [inline src/App.jsx:1]: would be best to leave the App.jsx file as decluttered as possible, you could create a component called Survey to import into this file, and use it as a parent component for Header, QuestionPage, StartPage etc.

**Review by AntonellaMorittu — APPROVED** [review]
hello ladies, you did a great job this week! Your code is clean, well formatted, the app satisfies the requirements and looks also very cute. I left a comment in the code, overall well done with responsiveness and accessibility, kudos! 👏

**Pull request closed without merging**